forest/sycamore/common.py

```diff
@@ -752,18 +764,15 @@ def read_aggregate_answers_stream(
             The first date of the survey data, in YYYY-MM-DD format
         time_end(str):
             The last date of the survey data, in YYYY-MM-DD format
+        history_path: Path to survey history file. If this is included, the
```

Comment:
Either style is acceptable but you should continue putting arg names on a separate line for consistency.